.github/workflows/update_dependencies_and_client.yml

```diff
@@ -115,3 +115,18 @@ jobs:
           flankScripts shell firebase checkForSdkUpdate \
           --github-token=${{ secrets.GITHUB_TOKEN }} \
           --zenhub-token=${{ secrets.ZENHUB_API_KEY }}
+
+  trigger-pointer:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ secrets.FLANK_RELEASE_APP_ID }}
+          private_key: ${{ secrets.FLANK_RELEASE_PRIVATE_KEY }}
```

Comment:
I truly get nervous about sharing private keys with opensource repositories.
Are we sure its trusted.

Reply:
The alternative is to create PAT and add it to secret. We use this logic in a couple of places inside the project. Actually, @piotradamczyk5 has configured this app and may has more detailed info

Reply:
This is the most popular workflow for this purpose and this is secure.
Github apps are better to manage than PAT, so that's the reason why we use them